README.md

```diff
@@ -114,22 +114,21 @@ After you're finished please remove all the comments and instructions!
 
 ## 🚀 Live Demo <a name="live-demo"></a>
 
-- [Live Demo Link](https://deploy-preview-31--rentbikes.netlify.app)
+- [Live Demo Link]
```

Comment:
_note: Since there is no open PR in the backend project for me to review, I'll include my backend review comments here._

# Backend project review comments

[ ] Kindly, include in your instructions the required db commands that your users will need to run your backend app: ``rails db:create, rails db:migrate, rails db:seed``. 👍